### PR TITLE
switches from sqlite to postgres input for uploader

### PR DESCRIPTION
**Pull request by djkapner**
General comments welcome. Of particular interest:
- [x] calling method, i.e. `sql_filter` is pretty wide open. (changed to `--roi_manifests_ids` arg)
- [x] @njmei Check out my tests mocking the dbconnection. Let me know if you see improvement areas. (see comments)

This command:
```
python -m slapp.transfers.upload \
    --roi_manifests_ids 1 2 3 \
    --s3_bucket_name danielk-ophys-dev \
    --prefix pr_demo
```
created objects in an S3 bucket:
image

and the contents of that `manifest.jsonl` are:
```
{"experiment-id": 840460366, "roi-id": 1575056, "source-ref": "s3://danielk-ophys-dev/pr_demo/20200428144657/outline_1575056.png", "roi-mask-source-ref": "s3://danielk-ophys-dev/pr_demo/20200428144657/mask_1575056.png", "video-source-ref": "s3://danielk-ophys-dev/pr_demo/20200428144657/video_1575056.mp4", "max-source-ref": "s3://danielk-ophys-dev/pr_demo/20200428144657/max_1575056.png", "avg-source-ref": "s3://danielk-ophys-dev/pr_demo/20200428144657/avg_1575056.png", "trace-source-ref": "s3://danielk-ophys-dev/pr_demo/20200428144657/trace_1575056.json"}
{"experiment-id": 840460366, "roi-id": 1575085, "source-ref": "s3://danielk-ophys-dev/pr_demo/20200428144657/outline_1575085.png", "roi-mask-source-ref": "s3://danielk-ophys-dev/pr_demo/20200428144657/mask_1575085.png", "video-source-ref": "s3://danielk-ophys-dev/pr_demo/20200428144657/video_1575085.mp4", "max-source-ref": "s3://danielk-ophys-dev/pr_demo/20200428144657/max_1575085.png", "avg-source-ref": "s3://danielk-ophys-dev/pr_demo/20200428144657/avg_1575085.png", "trace-source-ref": "s3://danielk-ophys-dev/pr_demo/20200428144657/trace_1575085.json"}
{"experiment-id": 840460366, "roi-id": 1575113, "source-ref": "s3://danielk-ophys-dev/pr_demo/20200428144657/outline_1575113.png", "roi-mask-source-ref": "s3://danielk-ophys-dev/pr_demo/20200428144657/mask_1575113.png", "video-source-ref": "s3://danielk-ophys-dev/pr_demo/20200428144657/video_1575113.mp4", "max-source-ref": "s3://danielk-ophys-dev/pr_demo/20200428144657/max_1575113.png", "avg-source-ref": "s3://danielk-ophys-dev/pr_demo/20200428144657/avg_1575113.png", "trace-source-ref": "s3://danielk-ophys-dev/pr_demo/20200428144657/trace_1575113.json"}
```

### CODE REVIEW

File: tests/transfers/test_upload.py [hunk 1]
```diff
@@ -1,42 +1,33 @@
...
-@pytest.fixture(scope='module')
-def db_file(tmpdir_factory):
-    db = tmpdir_factory.mktemp("upload").join("mydatabase.db")
-    conn = sqlite3.connect(db)
-    conn.execute("CREATE TABLE manifest_table (manifest text)")
-    conn.close
-    yield db
-
+@pytest.fixture
+def mock_db_conn_fixture(request, tmpdir_factory):
```
Comment (njmei) [inline tests/transfers/test_upload.py:10]: No need for `request` if you are not indirectly parametrizing this `mock_db_conn_fixture`.

Comment (djkapner) [reply tests/transfers/test_upload.py]: removed `request`

File: slapp/transfers/upload.py [hunk 2]
```diff
@@ -2,16 +2,10 @@
...
 
 class UploadSchema(argschema.ArgSchema):
-    sqlite_db_file = argschema.fields.InputFile(
-        required=True,
-        description="sqllite input db file")
-    sql_table = argschema.fields.Str(
-        required=False,
-        missing="manifest_table",
-        description="table where the manifests are stored")
     sql_filter = argschema.fields.Str(
```
Comment (djkapner) [inline slapp/transfers/upload.py:9]: @njmei and I discussed this filter as being too poorly-defined and possible subject to some unwanted postgres operations. We think changing to expect a list of `roi_manifests.id` and a harcoded SQL query `SELECT manifest FROM roi_manifests WHERE id IN {}` is better. This puts the burden on the upload user to figure out which `roi_manifests.id` values to upload.

Comment (djkapner) [reply slapp/transfers/upload.py]: made the change to the code and the PR description comments.

**Pull request merged**